lib/jnpr/junos/utils/sw.py

```diff
@@ -999,6 +999,8 @@ def _progress(report):
                         for x in self._RE_list
                         if re.search(r"(\d+)", x)
                     ]
+                    vc_members.remove(self.dev.facts["vc_master"])
```

Comment:
why are you removing and adding the same vc_master info  ?

Reply:
This is the fix to make sure vc_master gets installed at the end to handle unlink option on EX 3400 .

we just remove  the vc_master member from the list and  append it to the end of the vc_members list , 
so that vc_master will be installed at the end .
```
Netconf trace logs to check the order of installation .
root@nms5-ex4k-b:RE:1% cat /var/log/test.log | grep package-add
Sep 14 10:22:09 [NETCONF] - [84540] Incoming: <?xml version="1.0" encoding="UTF-8"?><nc:rpc xmlns:nc="urn:ietf:params:xml:ns:netconf:base:1.0" message-id="urn:uuid:5fded0f2-43a2-42e2-9623-c959072c9bc6"><request-package-add><no-validate/><package-name>/var/tmp/junos-install-ex-x86-64-21.4R3-S3.4.tgz</package-name><member>0</member></request-package-add></nc:rpc>]]>]]>
Sep 14 10:22:09 [NETCONF] - [84540] Incoming: <?xml version="1.0" encoding="UTF-8"?><nc:rpc xmlns:nc="urn:ietf:params:xml:ns:netconf:base:1.0" message-id="urn:uuid:fca1cc73-321e-42e0-bafc-11ceb6544a92"><request-package-add><no-validate/><package-name>/var/tmp/junos-install-ex-x86-64-21.4R3-S3.4.tgz</package-name><member>2</member></request-package-add></nc:rpc>]]>]]>
Sep 14 10:22:09 [NETCONF] - [84540] Incoming: <?xml version="1.0" encoding="UTF-8"?><nc:rpc xmlns:nc="urn:ietf:params:xml:ns:netconf:base:1.0" message-id="urn:uuid:d67b0c30-6b33-43cd-8fb5-6e79fe5998bf"><request-package-add><no-validate/><package-name>/var/tmp/junos-install-ex-x86-64-21.4R3-S3.4.tgz</package-name><member>1</member></request-package-add></nc:rpc>]]>]]>
Sep 14 10:32:01 [NETCONF] - [94770] Incoming: <?xml version="1.0" encoding="UTF-8"?><nc:rpc xmlns:nc="urn:ietf:params:xml:ns:netconf:base:1.0" message-id="urn:uuid:236e0899-2d9f-4a6d-8152-826b9f54ed1d"><request-package-add><no-validate/><package-name>/var/tmp/junos-install-ex-x86-64-21.4R3-S3.4.tgz</package-name><member>0</member></request-package-add></nc:rpc>]]>]]>
Sep 14 10:33:38 [NETCONF] - [94770] Incoming: <?xml version="1.0" encoding="UTF-8"?><nc:rpc xmlns:nc="urn:ietf:params:xml:ns:netconf:base:1.0" message-id="urn:uuid:b355af4e-a5b0-4335-91a6-48f99194342a"><request-package-add><no-validate/><package-name>/var/tmp/junos-install-ex-x86-64-21.4R3-S3.4.tgz</package-name><member>2</member></request-package-add></nc:rpc>]]>]]>
Sep 14 10:35:09 [NETCONF] - [94770] Incoming: <?xml version="1.0" encoding="UTF-8"?><nc:rpc xmlns:nc="urn:ietf:params:xml:ns:netconf:base:1.0" message-id="urn:uuid:c399439b-d77f-46a1-9c38-837b7f39f609"><request-package-add><no-validate/><package-name>/var/tmp/junos-install-ex-x86-64-21.4R3-S3.4.tgz</package-name><member>1</member></request-package-add></nc:rpc>]]>]]>
root@nms5-ex4k-b:RE:1% 
```